frame/pov-limit/src/weights.rs

```diff
@@ -0,0 +1,39 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! THIS IS NOT AUTO GENERATED!
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use sp_std::marker::PhantomData;
+
+/// Weight functions needed for pallet_pov_limit.
+pub trait WeightInfo {
+	fn hash_value() -> Weight;
+}
+
+/// Weights for pallet_pov_limit using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	fn hash_value() -> Weight {
+		// Minimum execution time: 35_358 nanoseconds.
+		Weight::from_ref_time(35_935_000 as u64)
```

Comment:
This is hardcoded. The benchmark should probably be executed on a parachain runtime that is going to be used for stress-testing Kusama. So the weight should be probably generated there. I don't think it makes sense to add this to the `kitchensink-runtime` so that we can have an accurate weight for this on substrate. @bkchr

Reply:
We need to deploy it to the kitchensink runtime and then add a test that the expected weight compared to the actual consumed one is not more than 1% difference or something like that.  
Otherwise we have no guarantee that it is working as intended.